packages/@react-types/shared/src/dnd.d.ts

```diff
@@ -155,8 +183,10 @@ export interface DraggableCollectionProps {
   onDragStart?: (e: DraggableCollectionStartEvent) => void,
   /** Handler that is called when the dragged element is moved. */
   onDragMove?: (e: DraggableCollectionMoveEvent) => void,
+  // TODO: is dropTarget being a dropTarget or Element too broad? Would be best if it could just be one or the other, but Element will be a thing
+  // if dropping on a non-collection drop target
   /** Handler that is called when the drag operation is ended, either as a result of a drop or a cancellation. */
-  onDragEnd?: (e: DraggableCollectionEndEvent) => void,
+  onDragEnd?: (e: DraggableCollectionEndEvent, dropTarget: DropTarget | HTMLElement, isInternalDrop: boolean) => void,
```

Comment:
Open to feedback on this. Technically the user could track where the dragged items were dropped, but feels like we could provide that to them instead

Reply:
seems better to include.

Reply:
I've actually got rid of `dropTarget` in `onDragEnd`, no need for it due to [b239afe](https://github.com/adobe/react-spectrum/pull/3266/commits/b239afe5d0fcf062f87123f7f947eba6e97da6b7). We can always bring it back in the future if needed